### PR TITLE
feat: add support for highlighting JSX

### PR DESCRIPTION
**Pull request by erunion**
## 🧰 What's being changed?

* [x] Adds support for JSX highlighting.

## 🧪 Testing

* [ ] Tests pass?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,6 +82,7 @@ const ele = syntaxHighlighter('console.log("Hello, world!");', 'js', { ...opts,
 | Java | `java` |
 | JavaScript | `coffeescript`, `ecmascript`, `javascript`, `js`, `node` |
 | JSON | `json` |
+| JSX | `jsx` |
```
Comment (rafegoldberg) [inline README.md:85]: We probably need to update the RDMD docs too!

**Review by rafegoldberg — COMMENTED** [review]
Change look good to me, but the build  seems to be failing on install?

**Review by julshotal — APPROVED** [review]
Looks good once CI succeeds 👍

File: README.md (hunk 1 above)
Comment (erunion) [reply README.md:85]: Will need to bump RDMD as well to pull in support for highlighting JSX codeblocks there too.

**Pull request merged**